cmd/vic-machine/create/create.go

```diff
@@ -1273,10 +1274,29 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 	return cakp.CertPEM, skp, nil
 }
 
-func logArguments(cliContext *cli.Context) {
+func (c *Create) logArguments(cliContext *cli.Context) []string {
+	args := []string{}
+
 	for _, f := range cliContext.FlagNames() {
 		if !cliContext.IsSet(f) {
-			continue
+			skip := true
+			// This is a workaround for cli.Context.IsSet() returning false when
+			// the short option for a cli.StringSlice is supplied instead of the long option
+			// see https://github.com/urfave/cli/issues/314
+			create := reflect.ValueOf(c).Elem()
+			for i := 0; i < create.NumField(); i++ {
```

Comment:
The reflection for Create structure can be done out of loop, to generate one map, and use map to check if that flag is set or not.